repos/system_upgrade/el7toel8/actors/satellite_upgrade_data_migration/actor.py

```diff
@@ -35,16 +35,27 @@ def process(self):
         # disable services, will be re-enabled by the installer
         for service_name in SERVICES_TO_DISABLE:
             for service in glob.glob(os.path.join(SYSTEMD_WANTS_BASE, '{}.service'.format(service_name))):
-                os.unlink(service)
+                try:
+                    os.unlink(service)
+                except Exception as e:
+                    self.log.warning('Failed disabling service {}: {}'.format(service, e))
```

Comment:
I didn't `return` here, as failing to disable those services won't fail the upgrade (they will just try to start before the installer deployed the new config, failing and producing useless errors in logs)